buck/tox_hooks.py

```diff
@@ -229,18 +297,49 @@ def customize_envconfig(self, config):
             config.allowlist_externals = ["{toxinidir}/rename.sh",
                                           "charmcraft"]
 
+def get_gitreview_file():
+    with open('.gitreview', 'r') as f:
+        contents = f.readlines()
+    return contents
+
+def get_gitreview_line(key):
+    for line in get_gitreview_file():
+        if line.split('=')[0] == key:
+            return line.split('=')[1].rstrip()
+
+def is_k8s_charm():
```

Comment:
to not rely on the name, we can read the metadata.yaml file , and check if the 'containers' key is defined or not.

Reply:
:+1: